templates/pages/reset-password.jsx

```diff
@@ -19,17 +19,17 @@ var ResetPassword = React.createClass({
       <div>
         <Header redirectText="Need an account?" redirectLabel="Sign up" redirectPage="signup" />
 
-        <div>
+        <div className="resetPasswordPage">
             {!this.state.submitForm && !this.state.email ?
               <RequestView submitForm={this.handleResetPassword}/> : false
             }
 
             {this.state.submitForm ?
               <IconText
-              iconClass="emailSentIcon fa fa-envelope-o"
-              className="emailSent centerDiv">
-                <p className="emailSentHeader">Check your email</p>
-                <p>We&#39;ve emailed you instructions for creating a new password.</p>
+                iconClass="emailSentIcon fa fa-envelope-o"
+                className="emailSent centerDiv">
+                  <p className="emailSentHeader">Check your email</p>
+                  <p>We&#39;ve emailed you instructions for creating a new password.</p>
```

Comment:
why would you ever escape a quote? This never needed htmlentities, not even in 1995...